src/main/java/org/terasology/launcher/LauncherInitTask.java

```diff
@@ -98,11 +95,11 @@ protected LauncherConfiguration call() {
 
             updateMessage(I18N.getLabel("splash_fetchReleases"));
             logger.info("Fetching game releases ...");
+            // implicitly fetches game releases and cache them
             final RepositoryManager repositoryManager = new RepositoryManager(client);
-            Set<GameRelease> releases = repositoryManager.getReleases();
 
+            // implicitly scans the game directory for installed games and cache them
             final GameManager gameManager = new GameManager(cacheDirectory, gameDirectory);
-            Set<GameIdentifier> installedGames = gameManager.getInstalledGames();
```

Comment:
do we call `getInstalledGames()` in other situations or can it be removed?

Reply:
Likewise, in the application controller:

https://github.com/MovingBlocks/TerasologyLauncher/blob/f5998e3168fe77dc78a5ee4b3e0077f313f219d0/src/main/java/org/terasology/launcher/ui/ApplicationController.java#L274-L276